mitodl/mdl-react-components
# Configured project for publishing

Fixes #3 

To test:

`docker-compose run web npm pack`

Then:
`tar tvf mdl-react-components-0.0.0.tgz`

should list the contents, you should only see `package.json`, the readme, license, and files in lib/
